docs-developer/deploying.md

```diff
@@ -50,6 +31,25 @@ git fetch upstream && git diff --name-only upstream/main..upstream/l10n | awk -F
 Be careful to always use the **create a merge commit** functionality, not
 *squash* or *rebase*, to keep a better history.
 
+## How to deploy main to production
```

Comment:
I think initially I put the l10n related stuff to below thinking that this is a less important step (or deployment step is more important since this file is dedicated to it). But also we want everyone to not forget this step, so makes sense to move it above.